### DISCUSSION TITLE
Traceable score multiplier

### DISCUSSION THREAD

**Discussion by the-wildguy** [ppy/osu#30002] [Q&A] [answered]
Traceable should have a score multiplier as well as affecting pp. The mod multipliers I am thinking of are 1.06 to match with hidden or 1.05 to match with new dt base mult

**Comment by Givikap120** [comment, marked as answer]
Traceable matching HD multiplier was agreed upon in multiple discussions, yes
As far as I know - this is planned for next multipliers update

**Reply by the-wildguy** [reply]
> Traceable matching HD multiplier was agreed upon in multiple discussions, yes As far as I know - this is planned for next multipliers update

Any idea if this will be retroactive

**Reply by Givikap120** [reply]
if multiplier will be raised - it's 100% would be retroactive
